script/fix-animation

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+import re
+import argparse
+
+
+def fix_animation(input: str, output: str) -> None:
+    with open(input) as f:
+        data = f.readlines()
+
+    output_data = []
+    for line in data:
+        if 'stylesheet' in line:
+            match = re.search(r'href="(.*?)"', line)
+            if match is None:
+                output_data.append(line)
+            else:
+                css_name = match[1].replace('\\', '/').split('/')[-1]
+                output_data.append(f'    <link rel="stylesheet" href="{css_name}">\n')
+        elif '<webots-view' in line:
+            match = re.search(
+                r'data-thumbnail=(.*?) .*data-scene=(.*?) .*data-animation=(.*?)>', line)
+            if match is None:
+                output_data.append(line)
+            else:
+                jpg_name = match[1].replace('\\', '/').split('/')[-1]
+                x3d_name = match[2].replace('\\', '/').split('/')[-1]
+                json_name = match[3].replace('\\', '/').split('/')[-1]
+                output_data.append(
+                    f'      <webots-view data-thumbnail={jpg_name} '
+                    f'data-scene={x3d_name} data-animation={json_name}></webots-view>\n',
+                )
+        else:
+            output_data.append(line)
+
+    with open(output, 'w') as f:
+        f.writelines(output_data)
+
+
+def main() -> None:
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument('animation')
+    parser.add_argument('output')
```

Comment:
For learnings: you might be interested in [`argparse.FileType`](https://docs.python.org/3/library/argparse.html#argparse.FileType), a class which can be passed to the `type` kwarg here. It handles opening the files as well as natively supporting using `-` for STDIN and or STDOUT.